src/parser/parser.yy

```diff
@@ -1682,10 +1681,8 @@ describe_edge_index_sentence
 
 rebuild_tag_index_sentence
     : KW_REBUILD KW_TAG KW_INDEX name_label {
-        auto sentence = new AdminJobSentence(meta::cpp2::AdminJobOp::ADD);
-        sentence->addPara("rebuild");
-        sentence->addPara("tag");
-        sentence->addPara("index");
+        auto sentence = new AdminJobSentence(meta::cpp2::AdminJobOp::ADD,
```

Comment:
the label is not used.

Reply:
name_label is used in sentence->addPara(*$4)